src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -166,6 +166,18 @@ define_themes! { [light:0, dark:1]
         }
         searcher {
             action_list_gap = 10.0, 10.0;
+            icons {
+                favorites = Rgba(0.98,0.584,0.122,1.0)  , Rgba(0.98,0.584,0.122,1.0);
+                io        = Rgba(0.475,0.494,0.145,1.0) , Rgba(0.475,0.494,0.145,1.0);
```

Comment:
what are favorites and io?

Reply:
`favorites` is used just for the "Favorites" icon. `io` is used for all icons in the input/output category.

Reply:
Ah, ok, makes sense. Hmm, I dont know why but I did not get it from the code. Maybe no change is needed.